specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -617,6 +869,41 @@
        "additionalProperties": true,
        "x-ms-external": true
     },
+    "DocumentLookupResult": {
+      "type": "object",
+      "properties": {
+          "Document": {
```

Comment:
For the sake of accuracy, I think you need to use `additionalProperties` and `x-ms-external: true` here, just like for `SearchDocument`. Even in the client programming model, there shouldn't be a `Document` property in the `Lookup` result.